crates/webidl/src/lib.rs

```diff
@@ -70,6 +72,45 @@ pub fn compile(webidl_source: &str) -> Result<String> {
     Ok(compile_ast(&ast))
 }
 
+/// ChromeOnly is for things that are only exposed to privileged code in Firefox.
+pub fn is_chrome_only(ext_attrs: &Vec<Box<ExtendedAttribute>>) -> bool {
+    ext_attrs.iter().map(|external_attribute| {
+        let mut ret = false;
+        match external_attribute {
+            box ExtendedAttribute::ArgumentList(al) => {
+                if al.name == "ChromeOnly" {
+                    ret = true;
+                    println!("ArgumentList");
+                }
+            },
+            box ExtendedAttribute::Identifier(i) => {
+                if i.lhs == "ChromeOnly" {
+                    ret = true;
+                    println!("Identifier");
+                }
+            },
+            box ExtendedAttribute::IdentifierList(il) => {
+                if il.lhs == "ChromeOnly" {
+                    ret = true;
+                    println!("IdentifierList");
+                }
+            },
+            box ExtendedAttribute::NamedArgumentList(nal) => {
+                if nal.lhs_name == "ChromeOnly" {
+                    ret = true;
+                    println!("NamedArgumentList");
+                }
+            },
+            box ExtendedAttribute::NoArguments(na) => {
```

Comment:
Add a starting underscore to a variable if it is unused. E.g. `na` becomes `_na` (you could also just use `_`).